github/actions_secrets.go

```diff
@@ -0,0 +1,130 @@
+// Copyright 2020 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+type PublicKey struct {
+	ID  *string `json:"key_id,omitempty"`
```

Comment:
Let's please keep the Go naming as close to the JSON naming as possible to be consistent with the rest of this repo.
So this would be `KeyID`, please.

Reply:
Done